uiowa/uiowa
# Update ighn.international to use uids_base

Relates to #1749

<img width="1461" alt="Screen Shot 2020-07-01 at 1 25 35 PM" src="https://user-images.githubusercontent.com/4663676/86278645-5e53d280-bb9e-11ea-86fc-45c9ee091608.png">


## To Test

- pull/composer/blt-frontend (you should see a new yarn workspace called ighn_core build)
- ds ighn.international.uiowa.edu
- go to https://internationalighn.local.drupal.uiowa.edu and switch the theme to uids_base
- see that that the view is "on brand" similar to the person content type.
